### PR TITLE
Publish application on gh-pages

### PR DESCRIPTION
**Pull request by leplatrem**
- [x] Fix relative/absolute paths
- [x] Add command to generate bundle
- [x] Provide basic instructions in readme

@ametaireau r?

### CODE REVIEW

**Comment by leplatrem** [comment]
Looks like we'll need some bits of #15 otherwise there's a polling on localhost:3000 in production :)

**Comment by almet** [comment]
It looks good to me. I'm not sure about the relation to #15, though.

**Comment by leplatrem** [comment]
After discussing with n1k0 the current configuration/approach is wrong.

We should output everything to the `build/` folder and use npm package gh-pages to push it through.

**Comment by leplatrem** [comment]
I updated the branch following @n1k0  advices.

It is now slick and clean, and modules are not so big (ref #15):

```
     Asset     Size  Chunks             Chunk Names
 bundle.js  5.75 kB       0  [emitted]  app
vendors.js   282 kB       1  [emitted]  vendors
   [0] multi vendors 40 bytes {1} [built]
```

I also replaced the set of git command by a simple call to `gh-pages` tool :)

@n1k0 r?

File: README.md [hunk 1]
```diff
@@ -48,7 +48,17 @@ Run both unit and browser tests:
...
-(*To be done*)
+In order to publish the application on [Github pages](https://pages.github.com/),
+you first need to build and minify the JavaScript files using:
+
+    $ npm run build
+
+Check the content of the `build/` folder, and then use the `gh-pages` utility
+to publish it into the `gh-pages` branch automatically:
+
+    $ ./node_modules/.bin/gh-pages --dist build/
```
Comment (n1k0) [inline README.md]: I wonder if we couldn't provide an `npm run gh-pages-publish` (name tbd) command to smoothen all this even further?

**Comment by n1k0** [comment]
Wow, many code removals, much useful, such convenience, so r+. Your call for the new command.

**Comment by leplatrem** [comment]
Cool :)

I updated the readme and added a simple `npm run publish` command. LGTM

**Comment by n1k0** [comment]
Makes me think I'll probably use the gh-pages stuff for pushing the kinto.js demo as well :)

**Pull request merged**